cg/cli/set.py

```diff
@@ -186,9 +188,58 @@ def microbial_order(context, apptag, order_id, user_signature):
         click.echo(click.style(f"order not found: {order_id}", fg='yellow'))
         context.abort()
 
-    for sample_obj in microbial_order_obj.microbial_samples:
-        context.invoke(microbial_sample, sample_id=sample_obj.internal_id,
-                       user_signature=user_signature, apptag=apptag)
+    if apptag:
+        for sample_obj in microbial_order_obj.microbial_samples:
```

Comment:
as this command changes a lot of samples at the same time, it would be recommended to add a "are you sure you want to change 3000 samples?" question. What do you think?

Reply:
I don't think changing samples was part of this PR, should we fix that in another PR?

Reply:
that sounds like a good idea!